### ISSUE TITLE
Fails to load with WACOM Graphire 2 Graphics Art TABLET ET-0405A-U

### ISSUE THREAD

**Issue by snovotill** [OpenTabletDriver/OpenTabletDriver#2377]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### Steps to Reproduce

Connect WACOM Graphire 2 Graphics Art TABLET ET-0405A-U and run OpenTabletDriver.
The driver detects the tablet and then barfs IndexOutOfRangeException:Error as shown below:

[Wacom ET-0405A-U:Info] Output mode: Absolute Mode
[Wacom ET-0405A-U:Info] Display area: [3328x1440@<1664, 720>:0°],
[Wacom ET-0405A-U:Info] Tablet area: [127.6x92.8@<63.8, 46.4>:0°],
[Wacom ET-0405A-U:Info] Clipping: Enabled
[Wacom ET-0405A-U:Info] Ignoring reports outside area: Disabled
[Wacom ET-0405A-U:Info] Tip Binding: [Mouse Button Binding: Left]@0%
[Settings:Info] Driver is enabled.
[IndexOutOfRangeException:Error]        Index was outside the bounds of the array.
   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosTabletReport..ctor(Byte[] report)
   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser.GetToolReport(Byte[] report)
   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser.Parse(Byte[] report)
   at OpenTabletDriver.Devices.DeviceReader`1.Main()

### Expected Behavior

Should just load and then the tablet should show in the GUI.

### Observed Behavior

Attempts to load but fails. See steps to reproduce.

### Tablet Device

WACOM Graphire 2 Graphics Art TABLET ET-0405A-U

### Diagnostic Information

{
  "App Version": "OpenTabletDriver v0.6.0.4",
  "Build Date": "5/4/2022",
  "Operating System": {
    "Platform": 2,
    "ServicePack": "",
    "Version": "10.0.22000.0",
    "VersionString": "Microsoft Windows NT 10.0.22000.0"
  },
  "Environment Variables": {
    "USER": null,
    "TEMP": "C:\\Users\\user\\AppData\\Local\\Temp",
    "TMP": "C:\\Users\\user\\AppData\\Local\\Temp",
    "TMPDIR": null,
    "USERPROFILE": "C:\\Users\\user"
  },
  "HID Devices": [
    {
      "DevicePath": "\\\\?\\usb#vid_046d&pid_0893&mi_04#8&e5688e2&0&0004#{dee824ef-729b-4a0e-9c14-b7117d33a817}",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "Logitech StreamCam",
      "SerialNumber": "B24BC445",
      "FriendlyName": "Logitech StreamCam",
      "VendorID": 1133,
      "ProductID": 2195,
      "InputReportLength": 0,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "\\\\?\\hid#vid_045e&pid_002b&mi_01&col02#8&6adee7e&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "Microsoft Internet Keyboard Pro",
      "SerialNumber": "",
      "FriendlyName": "Microsoft Internet Keyboard Pro",
      "VendorID": 1118,
      "ProductID": 43,
      "InputReportLength": 2,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "\\\\?\\hid#vid_045e&pid_002b&mi_01&col01#8&6adee7e&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "Microsoft Internet Keyboard Pro",
      "SerialNumber": "",
      "FriendlyName": "Microsoft Internet Keyboard Pro",
      "VendorID": 1118,
      "ProductID": 43,
      "InputReportLength": 3,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "\\\\?\\hid#vid_05e0&pid_1200#8&31b111b9&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\\kbd",
      "Manufacturer": "©Symbol Technologies, Inc, 2006",
      "ProductName": "Symbol Bar Code Scanner",
      "SerialNumber": "S/N:43A399D12FC42249A28F56B875522D30 Rev:NBRPUAAQ3",
      "FriendlyName": "Symbol Bar Code Scanner",
      "VendorID": 1504,
      "ProductID": 4608,
      "InputReportLength": 9,
      "OutputReportLength": 2,
      "FeatureReportLength": 5,
      "CanOpen": false
    },
    {
      "DevicePath": "\\\\?\\hid#vid_19f7&pid_001a&mi_03#8&1572b767&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "RØDE Microphones",
      "ProductName": "RØDE VideoMic NTG",
      "SerialNumber": "68815F50",
      "FriendlyName": "RØDE VideoMic NTG",
      "VendorID": 6647,
      "ProductID": 26,
      "InputReportLength": 10,
      "OutputReportLength": 10,
      "FeatureReportLength": 0,
      "CanOpen": true
    },
    {
      "DevicePath": "\\\\?\\hid#vid_045e&pid_002b&mi_00#8&1e717ac3&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\\kbd",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "Microsoft Internet Keyboard Pro",
      "SerialNumber": "",
      "FriendlyName": "Microsoft Internet Keyboard Pro",
      "VendorID": 1118,
      "ProductID": 43,
      "InputReportLength": 9,
      "OutputReportLength": 2,
      "FeatureReportLength": 0,
      "CanOpen": false
    },
    {
      "DevicePath": "\\\\?\\hid#vid_056a&pid_0011&col02#6&35c0d17a&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "WACOM",
      "ProductName": "ET-0405A-UV2.0-3",
      "SerialNumber": "",
      "FriendlyName": "ET-0405A-UV2.0-3",
      "VendorID": 1386,
      "ProductID": 17,
      "InputReportLength": 8,
      "OutputReportLength": 0,
      "FeatureReportLength": 2,
      "CanOpen": true
    },
    {
      "DevicePath": "\\\\?\\hid#vid_045e&pid_0040#7&2586aafb&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "Microsoft",
      "ProductName": "Microsoft 3-Button Mouse with IntelliEye(TM)",
      "SerialNumber": "",
      "FriendlyName": "Microsoft 3-Button Mouse with IntelliEye(TM)",
      "VendorID": 1118,
      "ProductID": 64,
      "InputReportLength": 5,
      "OutputReportLength": 0,
      "FeatureReportLength": 2,
      "CanOpen": false
    },
    {
      "DevicePath": "\\\\?\\hid#vid_056a&pid_0011&col01#6&35c0d17a&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "WACOM",
      "ProductName": "ET-0405A-UV2.0-3",
      "SerialNumber": "",
      "FriendlyName": "ET-0405A-UV2.0-3",
      "VendorID": 1386,
      "ProductID": 17,
      "InputReportLength": 5,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": false
    },
    {
      "DevicePath": "\\\\?\\hid#vid_046d&pid_0893&mi_05#9&13fd3cfb&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "Manufacturer": "Unknown Manufacturer",
      "ProductName": "Logitech StreamCam",
      "SerialNumber": "B24BC445",
      "FriendlyName": "Logitech StreamCam",
      "VendorID": 1133,
      "ProductID": 2195,
      "InputReportLength": 2,
      "OutputReportLength": 0,
      "FeatureReportLength": 0,
      "CanOpen": true
    }
  ],
  "Console Log": [
    {
      "Time": "2022-08-03T22:20:26.5532759-07:00",
      "Group": "Detect",
      "Message": "Searching for tablets...",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },

< S N I P >

    {
      "Time": "2022-08-03T22:20:26.554058-07:00",
      "Group": "Detect",
      "Message": "Searching for tablet 'Wacom ET-0405A-U'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5540639-07:00",
      "Group": "Device",
      "Message": "Initializing device 'ET-0405A-UV2.0-3' \\\\?\\hid#vid_056a&pid_0011&col02#6&35c0d17a&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5540699-07:00",
      "Group": "Device",
      "Message": "Using report parser type 'OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser'",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5540758-07:00",
      "Group": "Device",
      "Message": "Set device feature: 02-02",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5540822-07:00",
      "Group": "Detect",
      "Message": "Found tablet 'Wacom ET-0405A-U'",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },

< S N I P >

    {
      "Time": "2022-08-03T22:20:26.5545484-07:00",
      "Group": "Display",
      "Message": "DPI Awareness enabled",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5545565-07:00",
      "Group": "Wacom ET-0405A-U",
      "Message": "Output mode: Absolute Mode",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5545709-07:00",
      "Group": "Wacom ET-0405A-U",
      "Message": "Display area: [3328x1440@<1664, 720>:0°],",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5545768-07:00",
      "Group": "Wacom ET-0405A-U",
      "Message": "Tablet area: [127.6x92.8@<63.8, 46.4>:0°],",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5545831-07:00",
      "Group": "Wacom ET-0405A-U",
      "Message": "Clipping: Enabled",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.554589-07:00",
      "Group": "Wacom ET-0405A-U",
      "Message": "Ignoring reports outside area: Disabled",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5545948-07:00",
      "Group": "Wacom ET-0405A-U",
      "Message": "Tip Binding: [Mouse Button Binding: Left]@0%",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546007-07:00",
      "Group": "Settings",
      "Message": "Driver is enabled.",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546067-07:00",
      "Group": "IPC",
      "Message": "Connected to a client.",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546128-07:00",
      "Group": "IndexOutOfRangeException",
      "Message": "Index was outside the bounds of the array.",
      "StackTrace": "   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosTabletReport..ctor(Byte[] report)\r\n   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser.GetToolReport(Byte[] report)\r\n   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser.Parse(Byte[] report)\r\n   at OpenTabletDriver.Devices.DeviceReader`1.Main()",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546196-07:00",
      "Group": "Display",
      "Message": "DPI Awareness enabled",
      "StackTrace": null,
      "Level": 0,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546258-07:00",
      "Group": "Settings",
      "Message": "Presets have been refreshed. Loaded 0 presets.",
      "StackTrace": null,
      "Level": 1,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546321-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetManufacturer()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_Manufacturer>b__13_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546392-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_SerialNumber>b__19_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546461-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetManufacturer()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_Manufacturer>b__13_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546531-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_SerialNumber>b__19_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546598-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetManufacturer()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_Manufacturer>b__13_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546662-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_SerialNumber>b__19_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546727-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_SerialNumber>b__19_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546791-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_SerialNumber>b__19_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546856-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetSerialNumber()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_SerialNumber>b__19_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    },
    {
      "Time": "2022-08-03T22:20:26.5546921-07:00",
      "Group": "DeviceIOException",
      "Message": "Failed to get info.",
      "StackTrace": "   at HidSharp.Platform.Windows.WinHidDevice.RequiresGetInfo(GetInfoFlags flags)\r\n   at HidSharp.Platform.Windows.WinHidDevice.GetManufacturer()\r\n   at OpenTabletDriver.Devices.HidSharpBackend.HidSharpEndpoint.<>c.<get_Manufacturer>b__13_0(HidDevice d)\r\n   at OpenTabletDriver.Extensions.TryGet[TSource,TValue](TSource source, Func`2 predicate, TValue& value)",
      "Level": 3,
      "Notification": false
    }
  ]
}

**Comment by jamesbt365** [comment]
Could try making a `Configurations` folder next to the executables of otd and moving [this](https://github.com/OpenTabletDriver/OpenTabletDriver/files/9483558/ET-0405A-U.json.txt) here (remove the .txt on the end)

**Comment by snovotill** [comment]
Thanks, that helped a little bit in that the tablet is now being enumerated, but it still fails to function apparently due to other array out of bounds errors. Here is the debug log (attached).
[debug.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/9501365/debug.txt)

**Comment by jamesbt365** [comment]
The file i sent isn't using the intuos report parser so restart otd.

**Comment by snovotill** [comment]
Okay, so I rebooted my PC just be be sure and then I double-clicked on:
C:\ProgramData\OpenTabletDriver\OpenTabletDriver.UX.Wpf.exe

The output is a little different now but still includes an out of bounds error in the tail of the debug log as follows:
```
[Settings:Info]	Presets have been refreshed. Loaded 0 presets.
[Display:Debug]	DPI Awareness enabled
[IndexOutOfRangeException:Error]	Index was outside the bounds of the array.
   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosTabletReport..ctor(Byte[] report)
   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser.GetToolReport(Byte[] report)
   at OpenTabletDriver.Configurations.Parsers.Wacom.Intuos.IntuosReportParser.Parse(Byte[] report)
   at OpenTabletDriver.Devices.DeviceReader`1.Main()
[IPC:Debug]	Connected to a client.
[Settings:Info]	Driver is enabled.
[Wacom ET-0405A-U:Info]	Tip Binding: [Mouse Button Binding: Left]@0%
[Wacom ET-0405A-U:Info]	Ignoring reports outside area: Disabled
[Wacom ET-0405A-U:Info]	Clipping: Enabled
[Wacom ET-0405A-U:Info]	Tablet area: [127.6x92.8@<63.8, 46.4>:0°],
[Wacom ET-0405A-U:Info]	Display area: [3328x1440@<1664, 720>:0°],
[Wacom ET-0405A-U:Info]	Output mode: Absolute Mode
[Display:Debug]	DPI Awareness enabled
```

**Comment by snovotill** [comment]
Farther to above, I wanted to try an older version of the driver but had trouble installing the dependencies in Win11.

**Comment by X9VoiD** [comment]
Resolved by #2857
